### ISSUE TITLE
Create easy installation script to deploy the dotfiles

### ISSUE THREAD

**Issue by atxr** [goatfiles/dotfiles#12]
I'm trying to deploy the dotfiles in a lot of places and I definitly need a installation script.
I'm currently working on it, and will surely open a PR soon.

**Comment by amtoine** [comment]
> I'm trying to deploy the dotfiles in a lot of places and I definitly need a installation script.
>
yup definitely something important on the TODO list...
you can find inspiration in [`amtoine-install`](https://github.com/amtoine/scripts/blob/main/.deprecated/amtoine-install) :+1: 

> I'm currently working on it, and will surely open a PR soon.
>
i would love to see what you have in mind

would you like to share some ideas here before doing to much work and opening a PR?
you know, try to catch some things that could be improved right away, things that are great and things that should be removed :wink:

**Comment by atxr** [comment]
Basically I want to add an install script that
- clone the repo dotfiles and pkgbuilds
- builds the useful pkgbuilds
- install the dependencies in the toml file
- setup dmenu-flexipatch

What do you think about this?

**Comment by amtoine** [comment]
> * clone the repo dotfiles and pkgbuilds
>
do not forget about 
```bash
cfg config --local status.showUntrackedFiles no
cfg config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
cfg fetch --all --prune
```
and
```bash
cfg restore --staged
```
or
```bash
cfg reset --hard
```
> * builds the useful pkgbuilds
> * install the dependencies in the toml file
> * setup dmenu-flexipatch
> 
definitely all the main ingredients to having my config deployed:
(**1**) set the dotfiles in a *bare* repo,
(**2**) download the primordial packages from `amtoine/pkgbuilds`,
(**3**) install the rest of the dependencies from `pkgs.toml` and finally
(**4**) add the missing parts, e.g. the most important right now with `bakkeby/dmenu-flexipatch` :thumbsup:

> What do you think about this?
>
it looks like the most important things are there :thinking: 


looking forward to seeing this,
cant wait :star_struck:

**Comment by amtoine** [comment]
this has been closed in #13
